tests/crypto/mbedtls_psa/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(mbedtls)
+
+FILE(GLOB app_sources src/*.c)
+target_sources(app PRIVATE ${app_sources})
```

Not sure globbing is allowed. You may as well replace this by just
```suggestion
target_sources(app PRIVATE src/main.c)
```